crates/evm/core/src/backend/strategy.rs

```diff
@@ -181,6 +200,38 @@ impl BackendStrategyRunner for EvmBackendStrategyRunner {
     ) {
         EvmBackendMergeStrategy::merge_db_account_data(addr, active, fork_db);
     }
+
+    fn transact_from_tx(
+        &self,
+        back: &mut Backend,
```

Comment:
```suggestion
        backend: &mut Backend,
```
Or `db` are the idiomatic names.